.github/workflows/ci.yml

```diff
@@ -27,6 +27,9 @@ jobs:
             - name: Lint Files
               run: npm run lint
 
+            - name: Type Check Files
+              run: npm run tsc
```

Comment:
[Question] Does the team want to type check in CI separate from the build step? As in, was it intended that `npm run build` should be what type-checks?

Reply:
`npm run build` is intended to do everything in preparation for a package being published (Including `tsc`, running additional scripts, running Rollup, etc.).

If we want to highlight type checking errors, then I think it's fine to have as a separate CI check.

Reply:
Great. I've found it to be useful to have explicit checks for each of (lint, build, test, etc.) split out. What happens over time in larger repos is that type-checking time grows to be much much longer than build times, so folks reasonably want to be able to see results from tsc on its own separately from / in parallel to build. Both in CI and locally.